### PR TITLE
Add missing status property to Authenticators doc

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Description:
Added missing property `status` to `Authenticator` documentation [here](https://developer.okta.com/docs/reference/api/authenticators-admin/).

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md [hunk 1]
```diff
@@ -534,6 +534,7 @@ The Authenticator object defines the following properties:
 | `created`     | String (ISO-8601)                                               | Timestamp when the Authenticator was created                                   |
 | `id`          | String                                                          | A unique identifier for the Authenticator                                                |
 | `key`         | String                                                          | A human-readable string that identifies the Authenticator                                |
+| `status`      | `ACTIVE`,`INACTIVE`                                             | Status of the Authenticator                                |
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:537]: Ref: https://github.com/okta/okta-core/blob/master/components/core/factors/api/src/main/java/com/saasure/db/dto/core/entity/authenticators/Authenticator.java#L90-L93

Not sure if we need to document more possible statuses here.

Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:537]: Any other statuses?

Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:537]: Here's the answer from @SamSanjabi-okta: 

https://okta.slack.com/archives/C9D0TLFQ9/p1630447255069300?thread_ts=1630447011.068300&cid=C9D0TLFQ9

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**